src/sonic-yang-models/yang-models/sonic-neigh.yang

```diff
@@ -14,10 +14,14 @@ module sonic-neigh {
 	import sonic-portchannel {
 		prefix lag;
 	}
-	// TODO: Uncomment the following lines when sonic-vlan.yang is available
-	// import sonic-vlan {
-	// 	prefix svlan;
-	// }
+
+	import sonic-vlan {
```

Comment:
sonic-vlan has to be commented since there is yang bug which cause backlink issue